ui/apps/profile/src/components/pages/edit-profile/index.tsx

```diff
@@ -47,10 +47,7 @@ const EditProfilePage: React.FC<RootComponentProps & EditProfilePageProps> = pro
     },
   );
   const status = profileDataReq.status;
-  const { profile: profileData } =
-    profileDataReq.data && 'isViewer' in profileDataReq.data
-      ? profileDataReq.data
-      : { profile: null };
+  const { profile: profileData } = { ...{ profile: null }, ...profileDataReq.data };
```

Comment:
Make it more verbose by usign Object.assign, ex: `Object.assign({ profile: null }, profileDataReq.data)`